Source/WebCore/rendering/style/StyleImageSet.cpp

```diff
@@ -96,7 +96,7 @@ ImageWithScale StyleImageSet::bestImageForScaleFactor()
         result = image;
     }
 
-    if (!result.image)
+    if (!result.image || !result.scaleFactor)
```

Comment:
Should we assert that `result.scaleFactor` is never negative given calc() should have clamped it earlier?

Reply:
Yeah I'll add a debug assert here for that. Should catch any mistakes in the parser.